system-x/services/splunk/src/main/java/software/tnb/splunk/resource/local/LocalSplunk.java

```diff
@@ -29,7 +32,15 @@ public void deploy() {
             throw new IllegalStateException("HTTPS protocol is not implemented for Local Splunk service! Use HTTP.");
         }
         LOG.info("Starting Splunk container");
-        container = new SplunkContainer(image(), PORT, containerEnvironment());
+        Map<String, String> envs = containerEnvironment();
+        List<Integer> ports = new ArrayList<>();
+        ports.add(8000);
```

Comment:
can you please also create a constant for this port, something like `UI_PORT` or similar?